src/models/signer.cr

```diff
@@ -10,10 +10,10 @@ class PassKit::Signer
   @wwdr_cert : String
 
   def initialize
```

Comment:
Since these are pulled from the environment, replicating [something like this](https://github.com/place-labs/crystal-etcd/blob/master/src/etcd.cr#L12) would be more explicit.